packages/bundle/src/adaptiveCards/Attachment/AnimationCardAttachment.js

```diff
@@ -26,7 +26,13 @@ class AnimationCardAttachment extends React.Component {
   }
 
   render() {
-    const { adaptiveCards, attachment, attachment: { content: { media = [] } = {} } = {}, styleSet } = this.props;
+    const {
+      adaptiveCards,
+      adaptiveCardHostConfig,
```

Comment:
sort alphabetical

Reply:
You know... I thought I was.... thanks for pointing that out! 🙃